tests/Unit/DependencyInjection/ConfigurationTest.php

```diff
@@ -395,4 +395,22 @@ public function testCacheConfigMustHavePool()
         $file = __DIR__.'/../../Resources/Fixtures/config/cache_config_with_no_pool.yml';
         $this->assertProcessedConfigurationEquals([], [$file]);
     }
+
+    public function testLimitlessCapturedBodyLength()
+    {
+        $file = __DIR__.'/../../Resources/Fixtures/config/limitless_captured_body_length.yml';
+        $config = $this->emptyConfig;
+        $config['profiling']['captured_body_length'] = null;
+        $this->assertProcessedConfigurationEquals($config, [$file]);
+    }
+
+    /**
+     * @expectedException \Symfony\Component\Config\Definition\Exception\InvalidConfigurationException
+     * @expectedExceptionMessage The child node "captured_body_length" at path "httplug.profiling" must be an integer or null.
```

Comment:
please do not use this deprecated form to declare expected exceptions anymore. use `$this->expectException()` and `$this->expectExceptionMessage()` instead (right above the line that is epxected to throw the exception).

i am aware that in this case, the difference is irrelevant. but with longer tests, having the expectations on the method means confusion if an exception gets thrown before it was actually supposed to be triggered. also, we can use the ::class constant when expecting in the code, which is nicer.

Reply:
This change breaks test because the assert `expectException` and `expectExceptionMessage` are not available in PHPUnit 4.8.36 which is load to test the bundle with PHP 5.5.

Reply:
see https://travis-ci.org/php-http/HttplugBundle/jobs/503055333

Reply:
okay, sorry about that one. we really should bump the minimal version for this bundle :-/

but then its fine as is.